suitepicon/suitepicon-glyphs.scss

```diff
@@ -1,9 +1,10 @@
 @font-face {
```

Comment:
please move the font-face into the suiteicon.scss file. or at least update the file